paasta_tools/cli/cmds/validate.py

```diff
@@ -81,19 +81,49 @@ def get_schema(file_type):
     return json.loads(schema)
 
 
-def validate_schema(file_path, file_type):
-    """Check if the specified config file has a valid schema
+def validate_healthcheck_cmd_has_cmd(config_file_object):
+    """Check that healthcheck_cmd is included if healthcheck_mode
+    is set to cmd
+
+    :param config_file_object: the loaded config object
+    """
+    for config in config_file_object.values():
+        if 'healthcheck_mode' in config:
+            if config['healthcheck_mode'] == 'cmd':
+                assert 'healthcheck_cmd' in config,\
```

Comment:
That example actually looks better than the ones I found on SOF. I need to check it works with the healthcheck_mode itself being optional too. I'll give it another go, I'd definitely prefer if it was possible within the schema!